deployments/goerli/weth/configuration.json

```diff
@@ -20,7 +20,7 @@
   },
   "tracking": {
     "indexScale": "1e15",
-    "baseSupplySpeed": "0e15",
+    "baseSupplySpeed": "447916666666e0",
```

Comment:
I could modify the `stringToBigInt` to accept non-scientific notation numbers as well. But this is a quick workaround.